daemons/controld/controld_control.c

```diff
@@ -685,13 +685,6 @@ crmd_metadata(void)
     g_free(s);
 }
 
-static const char *
-controller_option(GHashTable *options, const char *name)
-{
-    return pcmk__cluster_option(options, controller_options,
-                                PCMK__NELEM(controller_options), name);
-}
```

Comment:
yeah I guess the only value this was adding is ensuring that name was in controller_options, which isn't worth it

Reply:
IIRC the result is NULL either way, with `g_hash_table_lookup()` versus with `pcmk__cluster_option()`.

Reply:
Yep but pcmk__cluster_option() will log an assertion

Reply:
Note I'm referring to name not being in controller_options, not name not being in options

Reply:
> assertion

Hmm, that is nice. If we decide we want to keep this for the assertion, it's not much trouble, we'd just have to make it non-static so that we can call it in the `controld_configure_X` functions. That was what I did originally before I decided `controller_option()` was redundant.

> name not being in controller_options

right

Reply:
The assertion isn't worth it. A better enhancement would be defined constants for all the option names, then it's a compile-time error